File: system-test/dotnet/Driver.Adapter/External/Erp/Client/Dtos/Error/ExtErpErrorResponse.cs
namespace Driver.Adapter.External.Erp.Client.Dtos.Error;

public class ExtErpErrorResponse
{
    public string? Message { get; set; }

    public static ExtErpErrorResponse From(string message)
    {
        return new ExtErpErrorResponse { Message = message };
    }
}

File: system-test/java/src/main/java/com/mycompany/myshop/testkit/driver/adapter/external/erp/client/dtos/error/ExtErpErrorResponse.java
package com.mycompany.myshop.testkit.driver.adapter.external.erp.client.dtos.error;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ExtErpErrorResponse {
    private String message;
}
